_assets/javascripts/components/pagination.js

```diff
@@ -83,4 +83,53 @@ CRDS.PageLoader.prototype.activatePage = function(type) {
 
 $(document).ready(function() {
   new CRDS.Pagination();
-});
\ No newline at end of file
+
+  const currentPage = parseInt(paginationInput.getAttribute("placeholder"));
+  const host = window.location.host;
+  const pageNumber = parseInt(paginationInput.value);
+  const paginationInput = document.getElementById("pagination-current-page");
+  const paginationNextButton = document.getElementById("pagination-next-button");
+  const paginationPrevButton = document.getElementById("pagination-previous-button");
+  const paginationSubmitButton = document.getElementById("pagination-submit-button");
+  const protocol = window.location.protocol;
+  const protocolAndHost = `${protocol}//${host}`
+  const totalPaginatedPages = parseInt(document.getElementById("total_pages").innerText);
+
+  paginationSubmitButton.addEventListener("click", function(e) {
+    if (pageNumber > 1 && pageNumber <= totalPaginatedPages) {
```

Comment:
It looks like this is the same block of code as starts at line `~110`. Can you abstract it to a function? I'd usually use [the rule of 3](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) before this but the code here is somewhat complex and rather only maintain it in one place.